src/Models.jl

```diff
@@ -49,6 +54,10 @@ return a `AbstractVector{<:Distribution}`.
 """
 function predict end
 
+function predict(model::Model, inputs::AbstractVector{<:AbstractVector})
+   return predict(model, hcat(inputs...))
```

Comment:
```suggestion
   return predict(model, reduce(hcat, inputs))
```

`reduce` is probably faster here, i guess because it can dispatch on `hcat` and hit a special method while also avoiding the cost of splatting. But i'm not exactly sure the reason. 
```julia
julia> using BenchmarkTools

julia> x = [rand(100) for _ in 1:100];

julia> @btime hcat($x...);
  17.019 μs (6 allocations: 79.11 KiB)

julia> @btime reduce(hcat, $x);
  6.182 μs (2 allocations: 78.20 KiB)
```